README.md

```diff
@@ -80,8 +84,8 @@ jobs:
 
 - `MABL_API_KEY` {string} - Your mabl API key
   [available here](https://app.mabl.com/workspaces/-/settings/apis) This should
-  be installed as a secret in your github repository.
-- `GITHUB_TOKEN` {string} (optional) - The Github token for your repository. If
+  be installed as a secret in your github repository. The key type should be "Command Line Interface"
+- `GITHUB_TOKEN` {string} (optional) - The Github token for your repository(Your uer icon > Settings > Developer Settings > Personal Access Token). If
```

Comment:
I think this token provided by GitHub to your repo workflows. Users shouldn't need to private a token here.
```suggestion
- `GITHUB_TOKEN` {string} (optional) - The GitHub token for your repository provided automatically by GitHub Actions. If
```